codespell_lib/data/dictionary.txt

```diff
@@ -5978,6 +5978,8 @@ esacpe->escape
 esacpes->escapes
 escapeable->escapable
 escapemant->escapement
+escpae->escape
+escpaed->escaped
```

Comment:
not highly likely but may also be `escapade` (link)](https://www.merriam-webster.com/dictionary/escapade)

Reply:
I think this is rare enough we should be okay either way